doc/source/ray-air/doc_code/predictors.py

```diff
@@ -0,0 +1,83 @@
+# flake8: noqa
+# isort: skip_file
+
+# __use_predictor_start__
+import numpy as np
+import tensorflow as tf
+
+import ray
+from ray.train.batch_predictor import BatchPredictor
+from ray.train.tensorflow import (
+    to_air_checkpoint,
+    TensorflowPredictor,
+)
+
+
+def build_model() -> tf.keras.Model:
+    model = tf.keras.Sequential(
+        [
+            tf.keras.layers.InputLayer(input_shape=(1,)),
+            tf.keras.layers.Dense(1),
+        ]
+    )
+    return model
+
+
+model = build_model()
+checkpoint = to_air_checkpoint(model)
+predictor = TensorflowPredictor.from_checkpoint(
+    checkpoint, model_definition=build_model
+)
+
+data = np.array([[1], [2], [3], [4]])
```

Comment:
Does this work if you flatten the array?

Reply:
done